### DISCUSSION TITLE
Statoin range.

### DISCUSSION THREAD

**Discussion by Taxom** [jvde-github/AIS-catcher#116] [Ideas]
Is it possible to make the maximum reception range is displayed not by sectors, but on the perimeter of the points of the maximum reception like in tar1090? This would give a greater resolution.
Thanks!

**Comment by jvde-github** [comment]
I am not familiar what they have done there but will have a look.

**Comment by Taxom** [comment]
As I understand it, the maximum reception distance is a figure drawn at the tops of the maximum distance points.
![Screenshot 2023-05-30 at 23-41-31 tar1090](https://github.com/jvde-github/AIS-catcher/assets/20313019/b14e0bb0-9ac8-438f-af24-cd3fd16e6a1c)
